src/Flagsmith.php

```diff
@@ -419,7 +419,7 @@ private function getIdentityFlagsFromApi(string $identifier, ?object $traits): F
         try {
             $data = IdentitiesGenerator::generateIdentitiesData($identifier, $traits);
             $apiFlags = $this->cachedCall(
-                'Global',
+                'Identity.'.$identifier,
```

Comment:
The cache key here should also include a hash of the traits for that user if provided.

Reply:
<img src="https://media3.giphy.com/media/pdSncNyYgaH0wqaCqp/giphy.gif"/>